internal/api/request_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package api
+
+import (
+	"bytes"
+	"net/url"
+	"testing"
+
+	"github.com/mattermost/genesis/model"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewCreateAccountRequestFromReader(t *testing.T) {
```

Comment:
Is there any particular reason for those methods being here? It seems like `model/account_request_test.go` would be better place for them.